astropy/tests/helper.py

```diff
@@ -126,10 +126,17 @@ def assert_follows_unicode_guidelines(x, roundtrip=None):
             assert eval(repr_x, roundtrip) == x
 
 
-@pytest.fixture(params=[0, 1, -1])
+@pytest.fixture(params=[0, 1, 4, -1])
 def pickle_protocol(request):
     """
-    Fixture to run all the tests for protocols 0 and 1, and -1 (most advanced).
+    Fixture to run all the tests for protocols 0, 1, 4 and 5 (most advanced).
```

Comment:
Since "5" is already mentioned in bullet point below under "-1", I think we should change this back to "-1" to be less confusing.
```suggestion
    Fixture to run all the tests for protocols 0, 1, 4 and -1 (most advanced).
```